README.md

```diff
@@ -40,6 +40,42 @@ For example:
     open-ce build env --python_versions=3.10 --ppc_arch=p10 --build_type=cpu python-env.yaml
 ```
 
+### Using Open-CE python with other Open-CE packages
+
+One can use Open-CE packages along with the optimized Open-CE Python. Following are the steps to setup Open-CE Python and Open-CE packages in a single conda environment -
+
+1. Create a new conda environment with optimized python built as mentioned in above section.
+```shell
+    conda create -n <conda env name> python=3.10.9 -c file://<path to condabuild directory>
+```
+   Here, `path to condabuild directory` means the path to `--output_folder` when python was built as per above section.
+   By default it is `condabuild` named directory.
```

Comment:
If you have not specified `--output-folder` , the directory will be `condabuild` .

Reply:
ok